Makefile

```diff
@@ -65,6 +68,10 @@ colima-all:
 	$(MAKE) colima-build
 	$(MAKE) colima-install
 
+colima-redo:
```

Comment:
What does this reset?

Reply:
I changed the name to `colima-deploy` that is doing build/install/restart.

If we believe it's not relevant I will remove it, let me know.